flink-runtime/src/main/java/org/apache/flink/runtime/execution/ExecutionState.java

```diff
@@ -25,16 +25,23 @@
  * <pre>{@code
  *
  *     CREATED  -> SCHEDULED -> DEPLOYING -> RUNNING -> FINISHED
- *                     |            |          |
- *                     |            |   +------+
- *                     |            V   V
- *                     |         CANCELLING -----+----> CANCELED
- *                     |                         |
- *                     +-------------------------+
+ *            |         |            |          |
+ *            |         |            |   +------+
+ *            |         |            V   V
+ *            |         |         CANCELLING -----+----> CANCELED
+ *            |         |                         |
+ *            |        +-------------------------+
+ *            |
+ *            |                                   ... -> FAILED
+ *           V
+ *    RECONCILING  -> RUNNING | FINISHED | CANCELED | FAILED
  *
- *                                               ... -> FAILED
  * }</pre>
  *
+ * <p>It is possible to enter the {@code RECONCILING} state from {@code CREATED}
```

Comment:
I think it would be best to move this paragraph below the other paragraphs below (below line 48).

A general note: I think for Javadocs it's enough to just have the opening `<p>` tag like this:

```
<p>Start here...
...continue and not closing tag at the end.
```